Tasks/IISWebAppManagementOnMachineGroupV0/task.json

```diff
@@ -17,7 +17,7 @@
     "version": {
         "Major": 0,
         "Minor": 5,
-        "Patch": 12
```

Comment:
u changed mange task instead of deploy task 😝

Reply:
Resolved